Ellielee308/wits-travel
# feat: upload form data to Firebase

[Website URL](https://witz-ec201--jane-2dqomh6w.web.app)

## Description
- The contact form uploads data to the Firestore form collection.
## Test Plan
- Submit the Form:
    - Enter content into the contact form and submit.
    - Verify that the successful upload message appears in the Google Developer Console after submission.
- Verify Data in Firestore:
    - Check the Firestore form collection to ensure that the form data has been correctly uploaded.
